src/cortex_cli/token_manager.py

```diff
@@ -32,13 +32,16 @@
     import daemon
 
 
-def daemonize_token_manager(cycle: int, config: ConfigFile, logfile: str = '/tmp/token_manager.log') -> None:
+def daemonize_token_manager(
+    cycle: int, config: ConfigFile, logfile: str = f'{Path.home()}/.iqm_cortex_cli/token_manager.log'
```

Comment:
IMHO it's not nice when a program pollutes in my home directory unnecessarily :) When possible, let's adhere to `XDG` standards. For caching it seems there is `$XDG_CACHE_HOME`.

> $XDG_CACHE_HOME defines the base directory relative to which user-specific non-essential data files should be stored. If $XDG_CACHE_HOME is either not set or empty, a default equal to $HOME/.cache should be used.

https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html

Especially since we already store tokens in `~/.cache/iqm-cortex-cli` by default.

Reply:
Ah, this seems more appropriate: 

>The $XDG_STATE_HOME contains state data that should persist between (application) restarts, but that is not important or portable enough to the user that it should be stored in $XDG_DATA_HOME. It may contain:
>- actions history (logs, history, recently used files, …)
>- current state of the application that can be reused on a restart (view, layout, open files, undo history, …)

> $XDG_STATE_HOME defines the base directory relative to which user-specific state files should be stored. If $XDG_STATE_HOME is either not set or empty, a default equal to $HOME/.local/state should be used.

Reply:
Sounds good, changed.

Reply:
Should we actually use the $XDG_STATE_HOME (and other corresponding XDG env vars, if applicable), and not just the default value?

Reply:
At least on Mac it is not available, of course we could check if it is available and set it to the default if it's not. What do you think?

Reply:
>if it is available and set it to the default if it's not

Yeah, that's the way XDG recommends to treat all their values.

Reply:
Changed :)